docs/index.md

```diff
@@ -31,64 +31,130 @@ You can easily integrate it with your existing tools, and feel lakeFS in action
+* **Maintaining auxilliary jobs that hydrate a development environment with data**
+
+If you execute any of these actions, adopting lakeFS will speed up development and deployment cycles, reduce the chance of incorrect data making it into production, and make it less painful in the event it does.
+
+Through its versioning engine, lakeFS enables the following built-in operations familiar from git:
+
+* **branch:** a consistent copy of a repository, isolated from other branches and their changes. Initial creation of a branch is a metadata operation that does not duplicate objects.
+* **commit:** an immutable checkpoint containing a complete snapshot of a repository.
+* **merge:** performed between two branches –– merges atomically update one branch with the changes from another.
+* **revert:** return a repo to the exact state of a previous commit.
+* **tag:** an immutable pointer to a single commits with a readable name.
+
+*The full list of commands available through the CLI can be [found here](./resources/commands.md).*
+
+Incorporating these operations into your data lake pipelines provides the same collaboration and organizational benefits you get when managing application code with source control.
+
+### The lakeFS promotion workflow
+
+To illustrate, we’ll show how lakeFS **branches** and **merges** improve the universal process of updating collections with the latest data.
+
+<img src="{{ site.baseurl }}/assets/img/promotion_workflow.png" alt="lakeFS promotion workflow" width="60%" height="60%" />
+
+1. First, create a new **branch** sourced from main to instantly generate a complete copy of your production data.
+2. Apply changes or make updates on the isolated branch to understand their impact prior to exposure.
+3. And finally, perform a **merge** from the feature branch back to main to atomically promote the updates into production.
+
+Following this pattern, lakeFS facilitates a streamlined data deployment workflow that consistently produces data assets you can have total confidence in.
+
+## What else does lakeFS do?
+
+lakeFS helps you maintain a tidy data lake in several other ways, including:
+
+#### Recovery from data errors
+Erroneous data that makes it into production is a newar inevitability given the complex and fast-moving nature of modern data pipelines. Similarly, critical data assets are liable to accidental deletion by poorly configured jobs or careless humans.
+
+Today, recovering from these events relies on periodic backups that 1) may be out of date and 2) require sifting through data at the object level. This process is inefficient and can take hours, days, or in some cases, weeks to complete.
+Recovering data in deletion events becomes an instant one-line operation with lakeFS using the ability to restore any historical commit.
+
+Reverting your data lake back to previous version using our command-line tool is explained [here](https://docs.lakefs.io/reference/commands.html#lakectl-branch-reset).
```

Comment:
```suggestion
Recovering data in deletion events becomes an [instant one-line operation](reference/commands.md#lakectl-branch-revert)  with lakeFS using the ability to restore any historical commit.
```
Note that the relevant command is "revert". Reset only rolls back uncommitted changes.